ConsoleApp/FoodCreator.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+
+namespace Snake
+{
+    class FoodCreator
+    {
+        int mapWidht;
```

Comment:
code convention violated here
should be 
```suggestion
        private int _mapWidht;
```